AltStore/Operations/FetchProvisioningProfilesOperation.swift

```diff
@@ -514,3 +523,13 @@ extension FetchProvisioningProfilesOperation
         }
     }
 }
+
+func containsNonASCII(text: String) -> Bool {
+    let ascii = CharacterSet(charactersIn: "\0"..."~")
+    for scalar in text.unicodeScalars {
+        if !ascii.contains(scalar) {
+            return true
+        }
+    }
+    return false
+}
```

Comment:
I think something like `text.allSatisfy { $0.isASCII }` would be better (leave the iteration and ASCII check to Swift)?

Reply:
Thank you for your suggestions. They made the code much cleaner. I will submit the PR again after making the changes. :)